subliminal/providers/subscenter.py

```diff
@@ -191,6 +192,7 @@ def query(self, title, season=None, episode=None):
                         subtitle_key = subtitle_item['key']
                         downloaded = subtitle_item['downloaded']
                         release = subtitle_item['subtitle_version']
+                        h_version = subtitle_item['h_version']
```

Comment:
It's some kind of a subtitle unique identifier which required as a param to get the subtitle.
See example here :
http://www.subscenter.org/he/cst/data/series/sb/greys-anatomy/13/22/